src/main/scala/nl.knaw.dans.easy.multideposit/parser/ProfileParser.scala

```diff
@@ -17,10 +17,10 @@ package nl.knaw.dans.easy.multideposit.parser
 
 import nl.knaw.dans.common.lang.dataset.AccessCategory
 import nl.knaw.dans.easy.multideposit.model._
+import nl.knaw.dans.lib.error._
```

Comment:
unused import

Reply:
when I remove import nl.knaw.dans.lib.error._ ,
'.combine(extract..........)' becomes red.
optimize imports also doesn't remove it, 
maybe we made some changes in extract functions

Reply:
I'm sorry, I thought this was just an unused import. What I didn't see at first in the diff on GitHub was that this import was actually moved (probably by doing a format) to it's correct place. It's good as it is right now.